lib/lists.js

```diff
@@ -1079,6 +1079,61 @@ exports.clear = function (bin) {
   return new ListOperation(opcodes.LIST_CLEAR, bin)
 }
 
+/**
+ * @summary Creates map create operation.
```

Comment:
```suggestion
 * @summary Creates list create operation.
```

Reply:
Corrected this in the most recent commit.